test/jdk/java/util/stream/BuiltInGatherersTest.java

```diff
@@ -245,34 +253,43 @@ public void testMapConcurrentAPIandContract(Config config) throws InterruptedExc
         }
```

Comment:
mapConcurrent is specified throw RuntimeException if the mapper throws. I read this to mean an Error is wrapped, and ambiguous on whether a runtime exception thrown by the mapper is wrapped or not. So I think this is actually tests, Error, RuntimeException and another runtime exception. This would help with the "Test cancellation after exception during processing" test too as it's not clear if TestException or a wrapping RuntimeException is correct.

Not important for now but testMapConcurrentAPIandContract could benefit from being split into a number of `@Test` methods. These tests aren't parameterized.